packages/mui-styles/src/makeStyles/makeStyles.test.js

```diff
@@ -13,28 +13,7 @@ import ThemeProvider from '../ThemeProvider';
 
 describe('makeStyles', () => {
   // StrictModeViolation: uses `useSynchronousEffect`
-  const mount = createMount({ strict: null });
-
-  /**
-   * returns a function that given the props for the styles object will return
-   * the css classes
-   * @param {object} styles argument for `makeStyles`
-   */
-  function createGetClasses(styles) {
-    const useStyles = makeStyles(styles);
-    const output = {};
-
-    function TestComponent(props) {
-      output.classes = useStyles(props);
-      return <div />;
-    }
-
-    return function mountWithProps(props) {
-      const wrapper = mount(<TestComponent {...props} />);
-      output.wrapper = wrapper;
-      return output;
-    };
-  }
```

Comment:
This custom setup has been replaced by using the `renderHook` directly. I think this makes tests easier to read by having less abstractions.